soc/renesas/smartbond/da1469x/Kconfig.defconfig

```diff
@@ -4,7 +4,7 @@
 if SOC_SERIES_DA1469X
 
 config SMARTBOND_TIMER
-	default y if PM
+	default y if PM && !$(dt_nodelabel_enabled,timer2)
```

Comment:
@ioannis-karachalios does PM need to be disabled in this case since systick will be the system timer ? it looks like PM is on by default and I don't know if it will go to sleep and not wake up

Reply:
@decsny PM is enabled by default (Smartbond is considered a low-power SoC) and requires that timer2 is not used as counter so that the OS tick can be generated (tick-less sleep mode). As such, **timer2 can only be used when PM is not used**.